### ISSUE TITLE
Not all transaction errors surfaced

### ISSUE THREAD

**Issue by Fang-** [urbit/bridge#227]
I deliberately put myself into a weird situation where I was using super-low gas prices for Ropsten. This caused a transaction error on-send, but that didn't get surfaced to the UI. Instead, (in the configure keys flow), it just gave me the option to "reset transaction" without telling me what or why.

<img width="876" alt="image" src="https://user-images.githubusercontent.com/3829764/62739622-19550980-b9ea-11e9-92df-ef078b4f0dbe.png">

**Comment by shrugs** [comment]
yup yup. no idea why this isn't being either 1) surfaced by web3 and/or 2) caught by our functions. web3 has famously not correctly throw errors in the past, but it'd be odd for that to have gone unsolved for so long.
